### DISCUSSION TITLE
Link (cross ref) to an element in a table cell?

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#4170] [Q&A] [answered]
Hello - inspired by how `\nameref{}` works in LaTeX, I'd like to do something like:

```markdown
+-----------------+------+
| Col1            | Col2 |
+:================+:=====+
| A {#tbl-refA}   | B    |
+-----------------+------+
| E               | F    |
+-----------------+------+
| A               | G    |
+-----------------+------+

: My Caption {#tbl-letters}

See @tbl-letters.

See @tbl-refA.

```

But it doesn't work -- the result of the second cross reference is

> See **?@tbl-refA**.

Whereas I'd like something like 

> See [A]()

with a hyperlink to the location in the table. 

Is something like that possible?

Is there some way to achieve this in Quarto?

**Comment by cscheid** [comment, marked as answer]
We don't currently have support for crossrefs to arbitrary portions of a file, because it's not clear how to make this work for arbitrary formats. The issue with your desired syntax is that we have names for all of our crossrefs: Table, Figure, Theorem, etc. At the very least, we'd have to figure out how a system to make something like this work in LaTeX. So, we'll keep it in mind, but no concrete estimate on when that'll happen.

**Reply by fuhrmanator** [reply]
No worries - I worked around it, but was wondering if I had missed an option. Cheers!!
